app/policies/participant_profile_policy.rb

```diff
@@ -1,6 +1,14 @@
 # frozen_string_literal: true
 
 class ParticipantProfilePolicy < ApplicationPolicy
+  def show?
+    return true if admin?
```

Comment:
This is just `admin_only`

Reply:
For now yes, but we also display participant profiles to SIC (which currently either does not use policy or use the old `ParticipantPolicy` which authorises User models. I have my doubts about `admin_only` method, I think `admin?` would be a much better name for it. `admin_only || record.some_condition` is a massive conceptual riddle